templates/home/index.html

```diff
@@ -52,7 +52,7 @@ <h1><span onclick="document.querySelector('.navbar a:first-child').click()"
                     <a class="nav-link" href="#">Team</a>
                 </li>
                 <li class="nav-item">
-                    <a class="nav-link" href="./">Alumni</a>
+                    <a class="nav-link" href="captures/">Captures</a>
```

Comment:
Use the templatetag `{% url 'home/captures' %}` here instead of mentioning the exact endpoint.